packages/flutter/lib/src/cupertino/split_form_row.dart

```diff
@@ -0,0 +1,199 @@
+  /// [CupertinoColors.destructiveRed] coloring, and is meant to inform the user
+  /// of issues attributed to the child widget. [errorText] is used in
+  /// [CupertinoTextFormField] to display validation errors.
+  const CupertinoSplitFormRow({
+    required this.leadingText,
+    this.helperText,
+    this.errorText,
+    required this.child,
+    Key? key,
+  }) : super(key: key);
+
+  /// Text that is shown leading the row.
+  ///
+  /// A [leadingText] is required. This string is displayed in a [Text]
+  /// widget leading to the side of the [CupertinoSplitFormRow]. It can be set
+  /// to an empty string, although it is encouraged to set the [leadingText] in
+  /// order to follow standard iOS practices.
+  final String leadingText;
+
+  /// String for text displayed underneath [leadingText] and [child]
+  ///
+  /// [helperText] appears in primary label coloring, and is meant to inform the
+  /// user about interaction with the child widget.
+  final String? helperText;
+
+  /// String for the error text displayed underneath the text widget and child.
+  ///
+  /// [errorText] appears in [CupertinoColors.destructiveRed] coloring and
+  /// medium-weight font, and is meant to inform the user of issues attributed
+  /// to the child widget. [errorText] is used in [CupertinoTextFormField] to
+  /// display validation errors.
+  final String? errorText;
+
+  /// Child widget trailing the row.
+  final Widget child;
+
+  @override
+  State<StatefulWidget> createState() => _CupertinoFormSplitRowState();
+}
+
+class _CupertinoFormSplitRowState extends State<CupertinoSplitFormRow> {
+  @override
+  Widget build(BuildContext context) {
+    final CupertinoThemeData themeData = CupertinoTheme.of(context);
+    final TextStyle textStyle = themeData.textTheme.textStyle;
+    final TextDirection currentDirection = Directionality.of(context);
+    final bool isRTL = currentDirection == TextDirection.rtl;
+    final List<Widget> rowChildren = <Widget>[
+      Padding(
+        padding: const EdgeInsetsDirectional.fromSTEB(10.0, 0.0, 1.0, 0.0),
+        child: Text(
+          widget.leadingText,
+          style: textStyle,
+        ),
+      ),
+      Flexible(
+        child: widget.child,
+      ),
+    ];
+
+    return DecoratedBox(
+      decoration: const BoxDecoration(
+        color: CupertinoColors.systemBackground,
+      ),
+      child: Padding(
+        padding: const EdgeInsetsDirectional.fromSTEB(6.0, 6.0, 6.0, 6.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: <Widget>[
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: isRTL ? rowChildren.reversed.toList() : rowChildren,
+            ),
+            if (widget.helperText != null && widget.helperText!.isNotEmpty)
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: <Widget>[
+                  Expanded(
```

Comment:
Maybe you just need Align. Generally shouldn't need to create Rows with just 1 item